intellij-settings/flows-config-schema.json

```diff
@@ -43,14 +33,30 @@
                 }
               }
             },
-            "subflow": {
+            "onPostAction": {
               "type": "string",
-              "description": "For screen within a subflow, they need the name of the subflow defined."
+              "description": "A method name to run on a POST."
```

Comment:
I think we should fill this description out a little more.   Something along the lines of "This is is the name of an Action method that will be called when a POST occurs, before any general data processing happens."

I think you can say Action here, so folks can link this back to what an Action is in the docs.